### ISSUE TITLE
New createAssayLink function 

### ISSUE THREAD

**Issue by lgatto** [rformassspectrometry/QFeatures#38]
See #36 for background.

**Comment by cvanderaa** [comment]
About your [comment](https://github.com/rformassspectrometry/Features/issues/36#issuecomment-578060944) in issue #36, what would you think of this?
```r
createAssayLink(object, ## Features object
                from, ## index or name of assay to link from
                to, ## index or name of assay to link to
                fcol)  ## the name of the shared feature variable in the respective rowData.
```
Splitting `i` in `from` and `to` will impose to the user to supply 2 arguments. I was thinking about calling them `i` and `j` to be consistent with the other functions, but this sounds less explicit.

**Comment by cvanderaa** [comment]
I think this can be closed since PR #56
